src/models/SeasonMetadata.ts

```diff
@@ -1,28 +1,10 @@
-import * as mongoose from 'mongoose';
-import { Schema, Model } from 'mongoose';
-import { CreditsResponse, TvSeasonImagesResponse, TvSeasonExternalIdsResponse } from 'moviedb-promise/dist/request-types';
-
-export interface SeasonMetadataInterface {
-  airDate?: string;
-  credits?: CreditsResponse;
-  externalIDs?: TvSeasonExternalIdsResponse;
-  images?: TvSeasonImagesResponse;
-  name?: string;
-  overview?: string;
-  posterRelativePath?: string;
-  seasonNumber: number;
-  tmdbID?: number;
-  tmdbTvID?: number;
-}
-
-// eslint-disable-next-line @typescript-eslint/no-empty-interface
-export interface SeasonMetadataModel extends Model<SeasonMetadataInterface> {}
+import mongoose, { Schema, InferSchemaType } from 'mongoose';
 
 const SeasonMetadataSchema: Schema = new Schema({
   airDate: { type: String },
-  credits: { type: Array },
-  externalIDs: { type: Array },
-  images: { type: Array },
+  credits: { type: Object },
```

Comment:
With the conversion to `InferSchemaType` here we have lost some types that were in the `interface`. To get back the existing types, instead of using `Object` types, these should be [subdocuments](https://mongoosejs.com/docs/subdocs.html)